hello-nginx-sample/README.md

```diff
@@ -0,0 +1,52 @@
+# Deploy multi-container service using nginx
+
+A Google Cloud Project is required in order to run the sample. The project should have the following API's enabled:
+
+* Cloud Run
+* Secret Manager
+
+## Enable required APIs
+
+```sh
+gcloud services enable secretmanager.googleapis.com run.googleapis.com
+```
+
+## Create new secret in Secret Manager
+
+* Going to the console https://console.cloud.google.com/security/secret-manager
+* Select `+ Create Secret` and name it `nginx_config` with the following secret value:
+
+```conf
```

Comment:
It might be nice to have this as a file and also provide a gcloud cmd: `gcloud secrets create nginx_config --data-file=FILE_NAME`. We may want to talk about test complexity ie do we create/delete this secret every test or have it as a long term resources. I think I'm fine either way.

Reply:
Given that the services/api's are enabled via gcloud, I think sticking to gcloud for creating a secret makes for a smoother user experience.